x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.ts

```diff
@@ -10,6 +10,15 @@ import { EventHit, TimelineEdges } from '../../../../../../common/search_strateg
 import { toStringArray } from '../../../../helpers/to_array';
 import { formatGeoLocation, isGeoField } from '../details/helpers';
 
+const getTimestamp = (hit: EventHit): string => {
+  if (hit.fields && hit.fields['@timestamp']) {
+    return (hit.fields['@timestamp'][0] ?? '') as string;
```

Comment:
disclaimer: haven't tested it, but I'm wondering if something like the following could eliminate the `as` cast here:

```
return `${hit.fields['@timestamp'][0] ?? ''}`;
```